src/jobs/deploy-s3.yml

```diff
@@ -1,9 +1,80 @@
-description: >
+description: |
   This job deploys an application to aws s3.
 
 executor: node/default
 
+parameters:
+  source:
+    type: string
+    default: SOURCE
+    description: |
+      a local directory path to sync with s3
+
+  s3_bucket:
+    type: string
```

Comment:
We may want to consider having this stored as an env_var_name and reference the context in the workflow instead of hard coding the value as a string